integration-tests/test-calico/bootstrap-single-subnet-aws.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+set -eux
+
+VPC_CIDR=172.30.0.0/24
+SUBNET0_CIDR=172.30.0.0/24
+
+alias aws="aws --output text"
+
+# Pre-deploy: Create a single-subnet VPC
+VPC_ID=$(aws ec2 create-vpc --cidr $VPC_CIDR | cut -f 7)
+SUBNET0_ID=$(aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block $SUBNET0_CIDR --availability-zone us-east-1b | cut -f 9)
+aws ec2 modify-subnet-attribute --subnet-id $SUBNET0_ID --map-public-ip-on-launch
+GATEWAY_ID=$(aws ec2 create-internet-gateway | cut -f 2)
+aws ec2 attach-internet-gateway --vpc-id $VPC_ID --internet-gateway $GATEWAY_ID
+ROUTE_TABLE_ID=$(aws --output text ec2 describe-route-tables | grep $VPC_ID | cut -f 2)
+aws ec2 create-route --route-table-id $ROUTE_TABLE_ID --destination-cidr-block 0.0.0.0/0 --gateway-id $GATEWAY_ID
+aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-support
+aws ec2 modify-vpc-attribute --vpc-id=$VPC_ID --enable-dns-hostnames
```

Comment:
Can you update this block to tag resources immediately after creating them?
```
VPC_ID=$(aws ec2 create-vpc --cidr $VPC_CIDR | cut -f 7)
aws ec2 create-tags --resource $VPC_ID --tags Key=created-by,Value=test-calico
SUBNET0_ID=$(aws ec2 create-subnet --vpc-id $VPC_ID --cidr-block $SUBNET0_CIDR --availability-zone us-east-1b | cut -f 9)
aws ec2 create-tags --resource $SUBNET0_ID --tags Key=created-by,Value=test-calico
...
```

I would do this for `VPC_ID`, `SUBNET0_ID`, `GATEWAY_ID` and `ROUTE_TABLE_ID`.

This way, if our cleanup step ever fails, we can visually identify orphaned resources and manually clean them up.

Reply:
Added the tags thanks for the pointers